src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -33,211 +33,230 @@
 public interface SdnRedirectionApi extends AutoCloseable {
 
     /**
-     * Create NetworkElement with a list of child elements
-     * For example, create NetworkElement with ports of protected workloads
-     * @param elements
-     *          ports of protected workloads or any network element
-     * @return the created NetworkElement
-     * @throws Exception
+     * Creates a NetworkElement
```

Comment:
Creates a network element
<p>
Describe what is a network element, i.e.: network port, a port group etc. You may need to look at some plugin implementations to come up with a description